client/src/app/interval-format.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'intervalFormat',
+})
+
+/**
+ * Pipe to format rule interval.
+ */
+export class IntervalFormatPipe implements PipeTransform {
+  /**
+   * Format interval to 'min' and 'hrs' respectively.
+   *
+   * @param {number} minutes
+   * @returns {string}
+   */
+  transform(minutes: number, ...args: unknown[]): string {
+    if (minutes <= 60) {
+      return `${minutes} min`;
+    } else {
+      return `${minutes / 60} hrs`;
```

Comment:
Do you need any rounding here or are you always expecting somewhat divisable numbers?
37 minutes => 0.616666666666...

Reply:
Since we are the ones setting the input, I didn't care much about validation.
If you think this is required I would surround the `else` minutes in `math.ceil()`.

Reply:
I think not doing any validation is going to be more obvious when we want to improve this down the road.